### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#55860]
#### VFS team name
VA Online Scheduling (VAOS)

#### VFS product name
VA Online Scheduling

#### VFS feature name
Appointment List

#### Point of Contact/Reviewers
Allison Christman - @allison0034 - Design

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
The "Start scheduling" button is navigation

### Link, screenshot or steps to recreate
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/42025-Allison-040702.png

### VA.gov Experience Standard
[Category Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
Navigation should be links and not buttons. Change to a link or action link

### Updated Design
[Figma Link](https://www.figma.com/file/FkNvnuYunT4tyAzQnCNM6l/Start-Scheduling-CTA?type=design&node-id=1522%3A22348&mode=design&t=DldhRJMRBNAcjjiZ-1)

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by ldelacosta** [comment]
@shiragoodman - we will review this ticket as part of Phase 2A MHV on VA.gov. We created a [ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/57151) for our team. cc: @JRRoof @klawyer @outerpress

**Comment by shiragoodman** [comment]
thanks @ldelacosta . Per our guidance above, we request that this ticket remain open until the issue has been resolved. Thanks for your understanding.

**Comment by ldelacosta** [comment]
We created a [FE ticket #64864](https://app.zenhub.com/workspaces/appointments-team-603fdef281af6500110a1691/issues/gh/department-of-veterans-affairs/va.gov-team/64864) to handle the work. cc: @outerpress @Cieramaddox @JRRoof

**Comment by outerpress** [comment]
@allison0034 @shiragoodman this work is complete. CC @JRRoof 

<img width="1417" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBLzlOQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--9652619bb46a2b4a6d2446d9a0eeba8e543b0264/Screenshot%202023-12-11%20at%205.14.28%E2%80%AFPM.png" alt="Screenshot 2023-12-11 at 5.14.28 PM.png" />

**Comment by shiragoodman** [comment]
Thanks @outerpress . You don't need our approval to close out the ticket, however if you'd like us to validate that the issue has been resolved, please let us know!

**Comment by outerpress** [comment]
Thanks @shiragoodman ! This one's pretty straightforward so I don't think we need validation. FYI @JRRoof closing this this ticket
